spec/system/tools_language_filter_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+describe 'filter[lang] for Tools Pages' do # rubocop:disable RSpec/DescribeClass
+  after(:all) { Zine.destroy_all } # rubocop:disable RSpec/BeforeAfterAll
+
+  before(:all) do # rubocop:disable RSpec/BeforeAfterAll
```

Comment:
Will this get you the same thing without needing to disable the Rubocop rule?

```suggestion
  before(:context) do
```

Reply:
gave it a shot, makes no difference https://github.com/crimethinc/website/runs/1920963201?check_suite_focus=true

Reply:
Gotcha. Bummer.